client/app/src/components/qr-scanner/qr-scanner.directive.js

```diff
@@ -63,7 +69,8 @@
         onSuccess: '&',
         onError: '&',
         onVideoError: '&',
-        inputCallback: '@'
+        inputCallback: '@',
+        inputCallbackFunc: '&'
```

Comment:
From what I can tell, this isn't used. Is that correct @Nasicus

Reply:
I'm not sure I understand: I'ts caled here: https://github.com/ArkEcosystem/ark-desktop/pull/456/files#diff-3671ac06b4eabc94ab421779e16417c1R28 (Line 28) and the "value" is set here: https://github.com/ArkEcosystem/ark-desktop/pull/456/files#diff-9ce1e6b4b80b3cd9b73da170356b437cR17

Reply:
My apologies, I was being thick! 👌